test/cbmc/proofs/Sntp_SendTimeRequest/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = Sntp_SendTimeRequest_harness
+
+# Please see test/cbmc/stubs/core_sntp_stubs.c for
+# more information on MAX_NETWORK_SEND_TRIES.
+MAX_NETWORK_SEND_TRIES=3
+
+# Maximum number of sntp time servers
+MAX_NO_OF_SERVERS=5
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = Sntp_SendTimeRequest
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=Sntp_SerializeRequest
```

Comment:
Abstracting this function is a proof assumption.  I'm having trouble understanding if this is a major assumption or not.  Maybe add a comment here advising the reader whether this something to think about or not.

You do define a stub for this in test/cbmc/stubs/core_sntp_stubs.c.  It is not clear to me how the other stubs you define are used since I don't see them invoked in your proof and it is unlikely something with the name ...Stub is invoked in the code being testing.

Reply:
Most of the stubs are for the interface that application needs to provide implementation for and I have sed this stubs in allocating SntpContext during Initializing of the library.